### PR TITLE
Github Calendar

### PR DESCRIPTION
**Pull request by dandeKim**
## description
- use a package react-calendar-heatmap for github-calendar
- use a package react-tooltip for react-calendar-heatmap
- add Legends

### CODE REVIEW

File: src/GitHubCalendar.js [hunk 1]
```diff
@@ -3,21 +3,80 @@
...
 
 class GitHubCalendar extends React.Component {
     constructor(props) {
         super(props);
-
-        this.state = {};
     }
 
+    countList = this.props.data.map((value) => value.count);
+    largest = Math.max.apply(null, this.countList);
```
Comment (soaple) [inline src/GitHubCalendar.js]: Please move this code to the inside of render function.

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

File: src/GitHubCalendar.js (hunk 1 above)
Comment (dandeKim) [reply src/GitHubCalendar.js]: I applied it!

**Review by soaple — APPROVED** [review]
LGTM (Looks Good To Me)

**Pull request merged**